tools/download-cleaner

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018 Ian Leonard (antonlacon@gmail.com)
+#
+# Scan through the given directory looking for out of date source packages.
+# If a source tarball is found that is different from what is currently in tree
+# print out the discovered files.
+#
+# If -d is passed to the script, also delete the files.
+
+set -e
+
+. config/options
```

Comment:
Use `. config/options ""` here, otherwise `$1`will be processed by `config/options`.